Cargo.toml

```diff
@@ -44,7 +44,8 @@ keccak = "0.1.2"
 # There is a proposal for extending nom::delimited to use this function:
 # https://github.com/Geal/nom/issues/1253
 parse-hyperlinks = { path = "./deps/parse-hyperlinks", version = "0.23.4" }
-felt = { package = "cairo-felt", path = "./felt", version = "0.1.0" }
+felt = { package = "cairo-felt", version = "0.1.1" }
```

Comment:
Keep the path for development. Otherwise we'll need to publish for every change or have ad-hoc changes.